pkg/extender/extender.go

```diff
@@ -576,6 +578,21 @@ func (e *Extender) processPrioritizeRequest(w http.ResponseWriter, req *http.Req
 	disableHyperconvergence := false
 	var err error
 
+	// Prioritize virt launcher pods on nodes to facilitate bind-mount
+	podPrefersBindMount := false
+	if e.Driver.PodPrefersBindMount(pod) {
```

Comment:
We are using PodPrefersBindMount and virt launcher nomenclature interchangeably and it is a bit confusing. Technically with hyperconvergence all pods prefer bind mounts. 

Instead lets make the Driver's API specific for VirtualMachine, for ex. IsVirtualMachineSupported() and then do the explicit check for ` pod.Labels["kubevirt.io"] == "virt-launcher"` here itself in extender.go

Reply:
Done